sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -183,7 +183,7 @@ namespace Azure { namespace Storage { namespace Blobs {
     /**
      * @brief Specifies the maximum number of containers to return.
      */
-    Azure::Core::Nullable<int32_t> MaxResults;
+    Azure::Core::Nullable<int32_t> PageSizeHint;
```

Comment:
How about `SinglePageSizeHint` to avoid misleading to page blob's size hint, which is not a thing.

Reply:
`PageSizeHint` is suggested by @tg-msft